.github/workflows/weekly_vuln_scan.yml

```diff
@@ -1,33 +1,50 @@
-# SPDX-FileCopyrightText: 2024 PNED G.I.E.
-# SPDX-FileContributor: Stichting Health-RI
-#
```

Comment:
important: you should put these comments back, otherwise the build will fail.

Reply:
Many thanks, I put related comments back in the yaml file. Now It needs only github token which can be defined in github secrets etc. It's up to you. Thanks again.